### PR TITLE
Salt Day 6 PR

### PR DESCRIPTION
**Pull request by Parzivus**

### CODE REVIEW

File: script/Arcueid/Day 6/06_00_ARC06_3_1.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ Though, perhaps it would be more accurate to say that it was also her boundary.
 [sha:3a3ad51a874d8e831887a34807a5b8baf1fd7eef]{
 -- Page 263, Offset 14547.
 -- 秋葉ひとりでは園まで出る事はなく、このテラスまでがあいつの移動範囲だった。
-She never went out into the garden by herself, this terrace marked the confines of where she was allowed to roam.
+Since she never went out into the garden by herself, this terrace marked the confines of where she was allowed to roam.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 6/06_00_ARC06_3_1.txt:52]: This isn't really causative here in JP, more two statements of fact - I guess you could change the comma to a semicolon if you wanted to make a change here?

File: script/Arcueid/Day 6/06_00_ARC06_3_1.txt [hunk 2]
```diff
@@ -225,7 +226,7 @@ That girl was nothing but trouble, but even so, it was... well, kinda fun to spe
 [sha:da7e29a27b4dbf7077cae9615fd32b77877855b8]{
 -- Page 276, Offset 14580.
 -- 一度殺されかけたっていうのに、何が未練なんだか。
-I nearly got killed, why on earth am I still pining after her?
+I nearly got killed, why on Earth am I still pining after her?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 6/06_00_ARC06_3_1.txt:229]: Earth only capitalized when talking about it as a celestial body 
https://www.dictionary.com/e/earth/

File: script/Ciel/Day 6/06_01_CIEL06_1.txt [hunk 3]
```diff
@@ -211,7 +211,8 @@ Resisting the temptation to abandon this train of thought, I take ten seconds to
 [sha:8d39d8369ea04082a441a2c956f9f0981fc07319]{
 -- Page 14, Offset 13189.
 -- ……どう考えても、この二日間の行動は弁明のしようがない。無断欠席。門限破り。おまけに音信不通ときた。
-No matter how much I rack my brain, I can't think of any way I could explain the events of the past two days...
+// Minor nitpick, but the phrasing makes it sound like absence, curfew, etc are explanations, rather than events. Threw in a rewrite to explain what I mean, thoughts/revisions?
+No matter how much I rack my brain, the events of the last two days seem impossible to explain...
```
Comment (rschlaikjer) [inline script/Ciel/Day 6/06_01_CIEL06_1.txt:215]: This changes the meaning a bit I think - it's not that the events can't be explained, it's that he can't explain _himself_. Maybe 
> [...] I can't work out how to explain away the events of the past two days...

File: script/Ciel/Day 6/06_01_CIEL06_1_1.txt [hunk 4]
```diff
@@ -131,6 +131,7 @@ I sit down on the sofa, doing my best to endure the pain running through my enti
 [sha:84255c3b15e88d7018db00d48a81719184862361]{
 -- Page 46, Offset 13269. Mods: @k, @e.
 -- 「卑怯でしょう。そんな顔をされたら気がゆる―――
+// "Of course" is not the correct response to "why." Requesting a JP speaker to suggest alternative to this line or the previous
```
Comment (rschlaikjer) [inline script/Ciel/Day 6/06_01_CIEL06_1_1.txt:134]: I guess you could change it to "isn't it obvious" or something? It drops the 'unfair' but it's in the spirit imo. Or rewrite more to something like
> [Huh?] How is it unfair?
> How isn't it unfair! If you...

File: script/Ciel/Day 6/06_01_CIEL06_1.txt (hunk 3 above)
Comment (yk1512) [reply script/Ciel/Day 6/06_01_CIEL06_1.txt:215]: I feel like using 'events' itself causes that slight shift in nuance. Making the subject himself like in 'I can't explain what I was up to in the past two days,' is probably more appropriate.

**Review by yk1512 — APPROVED** [review]

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 6/06_00_ARC06_3_1.txt (hunk 2 above)
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_3_1.txt:229]: Is that not what it's referring to here? I'm aware it's just a phrase and people aren't literally asking about planets or dirt, but this is in the same vein as "What in the world" and such.

Comment (rschlaikjer) [reply script/Arcueid/Day 6/06_00_ARC06_3_1.txt:229]: Nope, it's not a proper noun in expressions like down to earth, what on earth, etc.

**Pull request merged**